doc/cephadm/client-setup.rst

```diff
@@ -1,40 +1,41 @@
 =======================
 Basic Ceph Client Setup
 =======================
-Client machines need some basic configuration in order to interact with
-a cluster. This document describes how to configure a client machine
-for cluster interaction.
+Client machines requite some basic configuration to interact with
+Ceph clusters. This section describes how to configure a client machine
+so that it can interact with a Ceph cluster.
 
-.. note:: Most client machines only need the `ceph-common` package and
-          its dependencies installed. That will supply the basic `ceph`
-          and `rados` commands, as well as other commands like
-          `mount.ceph` and `rbd`.
+.. note:: 
+   Most client machines need to install only the `ceph-common` package
+   and its dependencies. Such a setup supplies the basic `ceph` and
+   `rados` commands, as well as other commands including `mount.ceph`
+   and `rbd`.
 
 Config File Setup
 =================
-Client machines can generally get away with a smaller config file than
-a full-fledged cluster member. To generate a minimal config file, log
-into a host that is already configured as a client or running a cluster
-daemon, and then run
+Client machines usually require smaller configuration files (here
+sometimes called "config files") than do full-fledged cluster members.
+To generate a minimal config file, log into a host that has been
+configured as a client or that is running a cluster daemon, and then run the following command:
 
-.. code-block:: bash
+.. prompt:: bash #
```

Comment:
nice to add a prompt here, but i think, the root privilege is not required to run `ceph` command line tool, isn't it?

Reply:
To be honest, I dunno. But I bet you do and you're right. So I've changed the prompt to a dollar sign.

Reply:
@zdover23 this comment is not addressed yet. did you forget to push the latest revision?